chromium_src/extensions/common/permissions/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
+  "+brave/components/skus/renderer",
```

Comment:
this is a violation.

Reply:
Removed - as noted above with move to `BraveExtensionsClient`